server.go

```diff
@@ -116,8 +119,55 @@ ______ _____  ___   _   __
 \_| \_\____/\_| |_/\_| \_/
                           
 ----------------------------
-Application should now be accessible on port `+os.Getenv("WEB_PORT")+`
+Application should now be accessible on port ` + os.Getenv("WEB_PORT") + `
 
 `
-    log.Println(logo)
+	log.Println(logo)
+}
+
+func cacheInit() {
+	config := bigcache.Config{
```

Comment:
some config are need to shift on .env but use default value here

Reply:
Will move the required values to env